rules/style.js

```diff
@@ -135,10 +158,13 @@ module.exports = {
 		"space-in-parens": 2,
 		// Require space around infix operators (e.g. var foo = 1 + 2, not var foo = 1+2)
 		"space-infix-ops": 2,
-		// Require spaces around unary words like void, new, delete, disallow around nonwords like ++, --, !
+		// Require spaces around unary words like void, new, delete,
+		// disallow around nonwords like ++, --, !
 		"space-unary-ops": 2,
 		// Require spaces in comments (e.g. // This is a comment, not //This is a comment)
 		"spaced-comment": [2, "always"],
+		// Never allow spaces within the curly braces of template strings
+		"template-curly-spacing": [2, "never"],
```

Comment:
For the most part we have no spaces here, but there are a few updates. Will auto-format!